src/README.md

```diff
@@ -20,9 +20,9 @@ Developer guides contain examples of code, mostly written in JavaScript and Soli
 
 ### Navigating
 
-* If you want an understanding of how **Unstoppable Domains work, its architecture and capabilities**, check out [General Overview](domain-registry-essentials/architecture-overview.md).
-* If you're a **developer** interested in **resolving records for cryptocurrency payments**, check out [Resolving Domain Records](domain-registry-essentials/resolving-domain-records.md).
-* If you're a **developer** interested in **how to resolve our domains in a browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
+- If you want an understanding of how **Unstoppable Domains work, its architecture and capabilities**, check out [General Overview](domain-registry-essentials/architecture-overview.md).
+- If you're a **developer** interested in **resolving records for cryptocurrency payments**, check out [Resolving Domain Records](domain-registry-essentials/resolving-domain-records.md).
+- If you're a **developer** interested in **how to resolve our domains in the browser**, check out [Resolving Domains in a Browser](browser-resolution/resolving-domains-in-a-browser.md).
```

Comment:
Is the indefinite article correct choice here? Right now it's not consistent with the name of the page